Web of Science Nextgen.js

```diff
@@ -2,14 +2,14 @@
 	"translatorID": "4a3820a3-a7bd-44a1-8711-acf7b57d2c37",
 	"label": "Web of Science Nextgen",
 	"creator": "Abe Jellinek",
-	"target": "^https://www\\.webofscience\\.com/",
+	"target": "^https://(www\\.)?webofscience\\..*/wos/",
```

Comment:
While false positives would be unlikely here, if there are known domains we should just use those (e.g., `^https://(www\\.webofscience\\.com/|webofscience\\.clarivate\\.cn)/`).